source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -105,11 +135,13 @@ export default class PinCode extends Component<Props> {
               themeId={IDENTIFIERS.INPUT}
```

Comment:
In the [line 29](https://github.com/input-output-hk/daedalus/pull/2470/files#diff-7277fe23c6cebe0ee1ba94c94700bdbc065de019a2268e29df9894a648fb4ccbR129), please make `id` unique or remove this pararm